apis/fluent-en_GB/extensions/jdk8/atrium-api-fluent-en_GB-jdk8-jvm/src/test/kotlin/ch/tutteli/atrium/api/fluent/en_GB/jdk8/ChronoLocalDateAssertionsSpec.kt

```diff
@@ -0,0 +1,11 @@
+package ch.tutteli.atrium.api.fluent.en_GB.jdk8
+
+import ch.tutteli.atrium.creating.Expect
+import ch.tutteli.atrium.domain.builders.creating.ChronoLocalDateAssertionsBuilder.isAfterOrEquals
+import ch.tutteli.atrium.specs.fun1
+import java.time.chrono.ChronoLocalDate
+
+object ChronoLocalDateAssertionsSpec : ch.tutteli.atrium.specs.integration.ChronoLocalDateAssertionsSpec(
+    fun1(Expect<ChronoLocalDate>::isAfterOrEquals) // Unresolved reference: isAfterOrEquals
```

Comment:
First of all, no worries, it's a good first issue I am not expecting that you see through the code at this stage.
You need to add an isAfterOrEquals method to the API as well (I thought I saw it before, have you removed it again?). Consult the to-do list in the issue description to check if other things are missing.